frontend/src/views/FuelSupplies/_schema.jsx

```diff
@@ -145,9 +145,9 @@ export const fuelSupplyColDefs = (optionsData, errors, warnings) => [
     minWidth: 250
   },
   {
-    field: 'fuelCategory',
+    field: 'fuelCategoryId',
```

Comment:
The value getter is still using return params.data.fuelCategory, why was this changed?

Reply:
Changed back.